lmk

```diff
@@ -592,6 +592,7 @@ def build_html_message_body(output, command=None, returncode=None,
             try:
                 line = line.decode('utf-8')
             except AttributeError:  # str.decode() depends on Python version.
+                line = line
```

Comment:
Also, since this code looks redundant, please add an inline comment to explain why it's necessary, e.g.

```
line = line  # Fix XXX error, issue #8.
```